source/voice_control/index.markdown

```diff
@@ -19,19 +19,33 @@ This section will help you set up Assist, which is Home Assistant voice assistan
 
 Assist allows you to control Home Assistant using natural language. It is built on top of an open voice foundation and powered by knowledge provided by our community.
 
-Assist is available to use on most platforms that can interface with Home Assistant. Look for the Assist icon <img src='/images/assist/assist-icon.svg' alt='Assist icon' style='height: 32px' class='no-shadow'>:
+Assist is available to use on most platforms that can interface with Home Assistant. Look for the Assist icon <img src='/images/assist/assist-icon.svg' alt='Assist icon' style='height: 32px' class='no-shadow'>
```

Comment:
We have a lot of single sentences as paragraphs. I would glue this one to previous paragraph.